source/Willow-Bootstrap/BootstrapComponentSupplier.class.st

```diff
@@ -83,8 +83,9 @@ BootstrapComponentSupplier >> checkboxUnlabeledOnModel: anObjectToUseWhenOn offM
 
 { #category : #Supplying }
 BootstrapComponentSupplier >> dateFieldApplying: aComponentCommand [
-
-	^ self singleLineTextFieldApplying: [ :field | (field setMaximumLengthTo: 10) + aComponentCommand + BootstrapDatepickerCommand new ]
+	
+	"The datepicker plugin don't plays nice with the HTML5 date input, so we use text input"
```

Comment:
Change comment to: "The datepicker plugin doesn't play nice with the HTML5 date input, so we use a text input instead."